WordPressEditor/build.gradle

```diff
@@ -4,10 +4,13 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:1.0.0'
+        classpath 'org.robolectric:robolectric-gradle-plugin:0.14.1'
```

Comment:
@tonyr59h this gradle plugin is deprecated. You might want to update `build:gradle` to at least `1.1` so that Robolectric can be integrated seamlessly.